advanced/Scripts/update.sh

```diff
@@ -49,14 +53,14 @@ GitCheckUpdateAvail() {
   REMOTE="$(git rev-parse @{upstream})"
 
   if [[ ${#LOCAL} == 0 ]]; then
-    echo "::: Error: Local revision could not be obtained, ask Pi-hole support."
-    echo "::: Additional debugging output:"
+    echo -e "        ${COL_LIGHT_RED}Error: Local revision could not be obtained, ask Pi-hole support."
```

Comment:
Just double checking with you, but is the 8 space indenting on error lines like 56/57/62/63/101/102/133/134/180/194 intentional?